core/src/main/java/org/eclipse/krazo/Properties.java

```diff
@@ -47,18 +44,12 @@ public interface Properties {
     String REDIRECT_SCOPE_COOKIE_NAME = "org.eclipse.krazo.redirectScopeCookieName";
 
     /**
-     * The implementation of {@link org.eclipse.krazo.security.CsrfTokenStrategy}
-     * to use for storing tokens.
+     * The implementation of {@link org.eclipse.krazo.security.CsrfTokenStrategy} to use for storing tokens.
      */
     String CSRF_TOKEN_STRATEGY = "org.eclipse.krazo.csrfTokenStrategy";
 
     /**
      * Property for defining default file extension for usage in views
      */
     String DEFAULT_VIEW_FILE_EXTENSION = "org.eclipse.krazo.defaultViewFileExtension";
-    
-    /**
-     * Boolean property which enables the {@link HiddenMethodFilter} when set to <code>true</code>.
-     */
-    String HIDDEN_METHOD_FILTER_ACTIVE = "org.eclipse.krazo.hiddenMethodFilterActive";
```

Comment:
I'm wondering if we can find a way to preserve the old behavior for at least some time for allowing people easier migration!?! Just thinking out loudly. Or do you think that providing some hints regarding migration would be sufficient?

Reply:
I think providing some hint in the release notes is enough. TBH, I think this feature wasn't used that much, because before the spec change it was an opt-in. So if you used it before, nothing changes.

Reply:
Ok. Sure. I'm fine with this as well. We just have to make sure to not forget about the documentation...